src/datreant/tests/test_treants.py

```diff
@@ -367,14 +367,26 @@ def test_add_categories(self, treant):
             treant.categories.add({'bark': 'snark'}, lark=27)
             assert 'bark' in treant.categories
             assert 'snark' not in treant.categories
-            assert 'bark' in treant.categories
+            assert 'lark' in treant.categories
 
             assert treant.categories['bark'] == 'snark'
             assert treant.categories['lark'] == 27
 
             treant.categories['lark'] = 42
             assert treant.categories['lark'] == 42
 
+            treant_tuple = ('ark', 'clark', 1)
```

Comment:
This isn't entirely your fault, but this test is long and should be split up (ie setting an int category doesn't affect setting a tuple category).  Can you rewrite this as a `parametrize` over the different input types?

Reply:
Sure, can do!